resources/auth.py

```diff
@@ -244,6 +244,7 @@ def post(self):
                 description: When set password token is no longer valid
         """
         MIN_PASSWORD_LENGTH = 8
+        MAX_PASSWORD_LENGTH = 72 #UTF-8 bytes
```

Comment:
E261, at least two spaces before inline comment
```suggestion
        MAX_PASSWORD_LENGTH = 72  #UTF-8 bytes
```